packages/patternfly-4/react-core/src/components/Checkbox/Checkbox.js

```diff
@@ -24,6 +26,7 @@ const defaultProps = {
   className: '',
   isValid: true,
   isDisabled: false,
+  isChecked: false,
```

Comment:
I am definitely ok with moving to I checked, but setting this to false will break uncontrolled versions. This needs to not be in default props. Lint will get mad so maybe try undefined or null here. Not sure which will work.